local/hack.go

```diff
@@ -1,26 +1,69 @@
 package main
 
 import (
+	"context"
+	"fmt"
+	"os"
 	"path/filepath"
 
-	"github.com/stackrox/rox/central/cve/datastore"
+	"github.com/stackrox/rox/central/cve/image/datastore"
+	"github.com/stackrox/rox/central/globaldb"
 	"github.com/stackrox/rox/central/globalindex"
 	"github.com/stackrox/rox/central/option"
 	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/pkg/postgres/pgconfig"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/search"
 )
 
 func main() {
 	if features.PostgresDatastore.Enabled() {
-		return
-	}
 
-	option.CentralOptions.DBPathBase = "local/database-restore/full"
+		// PREREQUISITES TO RUNNING POSTGRES
+		// 1. Get a SQL dump of DB that has the data you want]
+		//    kubectl exec -ti -n=stackrox $(kubectl get pod -n=stackrox -l app=central-db --output 'jsonpath={.items..metadata.name}') -- sh -c '/usr/bin/pg_dumpall' > local/central_active.sql
+		// 2. Run a docker container with postgres:
+		//   docker run -it -p 5432:5432 -e POSTGRES_USER=local-postgres -e POSTGRES_PASSWORD=local-pg-password postgres:14.2-alpine
+		// 3. Restore the SQL dump from #1 to this local DB
+		//   cat local/central_active.sql |  docker exec -i $(docker ps -lq) sh -c 'PGPASSWORD=local-pg-password psql -h localhost -U local-postgres -p 5432'
+		// 4. Go to pkg/config/config.go, and update the variable "defaultDBSource" to:
+		//		host=localhost port=5432 user=local-postgres sslmode=disable statement_timeout=600000 pool_min_conns=1 pool_max_conns=90 client_encoding=UTF-8
```

Comment:
This would be a whole lot easier if the default config was a exported variable, but I'm not sure what risks that exposes. Any thoughts?

Reply:
You can check configmap central-config, in central-config.yaml centralDB.source

Reply:
Oh, you do not have those.

Reply:
Nope this is run locally (ie via IntelliJ or with go run) so no configmap is available